app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/declaration/Declaration.scala

```diff
@@ -108,7 +109,9 @@ case class DeclarationJourney(sessionId: SessionId,
                               maybeName: Option[Name] = None,
                               maybeAddress: Option[Address] = None,
                               maybeEori: Option[Eori] = None,
-                              maybeJourneyDetails: Option[JourneyDetails] = None)
+                              maybeJourneyDetails: Option[JourneyDetails] = None,
+                              maybeGoodsDestination: Option[GoodsDestination] = None,
+                              maybeValueWeightOfGoods: Option[Boolean] = None)
```

Comment:
When I read 'maybeValueWeightOfGoods' I wasn't sure which of true or false meant the user should make a declaration.

Perhaps rename to 'maybeValueWeightOfGoodsExceedsThreshold'?